remoteworkitem/remoteworkitem.go

```diff
@@ -66,13 +66,7 @@ type AttributeAccesor interface {
 	Get(field AttributeExpression) interface{}
 }
 
-// RemoteWorkItem is the Database stored TrackerItem
-type RemoteWorkItem struct {
-	ID      string
-	Content []byte
-}
-
-var RemoteWorkItemImplRegistry = map[string]func(RemoteWorkItem) (AttributeAccesor, error){
+var RemoteWorkItemImplRegistry = map[string]func(TrackerItem) (AttributeAccesor, error){
```

Comment:
Thanks @tsmaeder , tracking this issue as part of #262 
Added the fix to this PR.